release-notes/opensearch-release-notes-1.3.0.md

```diff
@@ -3,7 +3,7 @@
 ## Release Highlights
 
 ### OpenSearch Ml Commons
-* ml-commons now officially becomes part of the OpenSearch 1.3.0 release.
+* The new ML Commons plugin empowers users to train and apply machine learning models as a part of the OpenSearch 1.3.0 release.
```

Comment:
@ylwu-amzn @Naarcha-AWS @stockholmux Please review this release highlight for ml-commons.
Thanks!

Reply:
LGTM. Thanks @gaiksaya